beaker-module_install_helper.gemspec

```diff
@@ -28,5 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec'
 
   # Run time dependencies
-  spec.add_runtime_dependency 'beaker', '>= 2.0'
+  spec.add_runtime_dependency 'beaker', '= 2.52' if RUBY_VERSION <= '2.1.6'
+  spec.add_runtime_dependency 'beaker' if RUBY_VERSION > '2.1.6'
```

Comment:
So I should be moving this dependency to the Gemfile and updating the version correctly?